packages/dashboard/src/pages/Settings/components/SettingSelect.tsx

```diff
@@ -0,0 +1,12 @@
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+import { Select, experimentalStyled as styled, selectClasses, outlinedInputClasses } from '@material-ui/core'
+
+export default styled(Select)(() => ({
+    [`& .${selectClasses.select}.${selectClasses.outlined}`]: {
+        padding: '12.5px 70px 12.5px 14px !important',
```

Comment:
please use the `theme.spacing`

Reply:
Actually, I have a doubt about this now. 🤔 If we're not using the dynamic spacing and the design is not following the dynamic spacing system, why we should stick on those...

Reply:
Ok, if we're not using dynamic spacing, We really don't need to stick on these